apps/web/modules/ee/sso/lib/utils.ts

```diff
@@ -1,5 +1,35 @@
+import { redactPII } from "@/modules/ee/audit-logs/lib/utils";
+import type { Account } from "next-auth";
+import { TUser } from "@formbricks/types/user";
+
 export const getCallbackUrl = (inviteUrl?: string, source?: string) => {
   return inviteUrl
     ? `${inviteUrl}${inviteUrl.includes("?") ? "&" : "?"}source=${source}`
     : `/?source=${source}`;
 };
+
+const sanitizeUrl = (url: string): string => {
```

Comment:
You introduce these functions in `ee/sso` but for me they seem to be generic for logging and we will also use them in other places for logging.
If we have them in the `ee` directory, we "cannot" use them in the community edition, but I would argue that this is like the logger itself a community edition feature.
So we should have them either in a generic place in the apps/web if they are specific to the nextjs app or in the logger package itself.